pace-util/Makefile

```diff
@@ -51,28 +51,15 @@ lint:
 	mypy --follow-imports silent --show-error-codes fv3gfs
 	@echo "LINTING SUCCESSFUL"
 
-# We do not use --no-annotate to keep the dependencies annotations
```

Comment:
pace-util uses the top level constraints file via a symbolic link, this target would overwrite that symlink and should be removed.

Reply:
I was wrong - this must be in a branch. Reverting this removal.